### DISCUSSION TITLE
Unable to download source code

### DISCUSSION THREAD

**Discussion by lingampe** [community/community#43923] [New to GitHub]
I have trigger automation from CAF, but my project source code unable to download facing access issue, for reference find the attachment.
  
![image](https://user-images.githubusercontent.com/104564986/211710841-bf6afead-b1d4-4b02-8f5f-029a23db5fc8.png)

**Comment by yiqing-zhao** [comment]
Hello @lingampe, it seems that you will need to re-authorize the OAuth Application `Intel Devtool`, 

Revoking the current application in the **Github Settings** > Settings > [Applications](https://github.com/settings/applications) > Authorized OAuth Apps
, and then try the `git pull` again.
